config/src/config.rs

```diff
@@ -325,6 +325,8 @@ pub struct Mining {
     pub data_request_timeout: Duration,
     /// Genesis block path
     pub genesis_path: String,
+    /// Binary flag to create a mint with a split reward
+    pub split_mint: bool,
```

Comment:
Why not putting the value of the outputs itself and defaulting to whatever is the block reward?

Reply:
I include documentation in witnet.toml explaining it